src/app/services/authentication/authentication.service.ts

```diff
@@ -36,19 +36,19 @@ export class AuthenticationService {
   }
 
   loginResponse(response: Login): void {
-    console.log("Login Response : " + response);
+    console.log("Login Response : " + JSON.stringify(response));
```

Comment:
Why is this log statement required

Reply:
It prints the content of the object in JSON string. The current print out show `[object]`?

Reply:
We know this functionality works. I am just confused why we are keeping this log statement that does little to tell us anything about how the system is running

Reply:
Removed https://github.com/usdot-fhwa-OPS/v2xhub-angular-ui-prototype/pull/1/commits/003442118db552098f6bd39ddcda27bf66a91bec